src/hyperloglog.c

```diff
@@ -464,7 +557,11 @@ int hllPatLen(unsigned char *ele, size_t elesize, long *regp) {
      *
      * This may sound like inefficient, but actually in the average case
      * there are high probabilities to find a 1 after a few iterations. */
+#ifndef USE_WYHASH
     hash = MurmurHash64A(ele,elesize,0xadc83b19ULL);
```

Comment:
Since the MutmurHash64A is only used in hyperloglog, the hash data quality only affect the estimate accuracy, from estimate accuracy perspective, in theory we may expect wyhash can increase the estimate accuracy compared with MutmurHash64A according to the smasher test report. 
From the performance perspective, we already see the perf gain https://github.com/redis/redis/issues/12912.